src/api/spec/services/meta_controller_service/project_updater_spec.rb

```diff
@@ -0,0 +1,91 @@
+require 'rails_helper'
+
+RSpec.describe ::MetaControllerService::ProjectUpdater do
+  let(:project) { create(:project, name: 'openSUSE_41') }
+  let(:admin_user) { create(:admin_user, login: 'Admin') }
+
+  before do
+    User.current = admin_user
+  end
+
+  context 'with valid meta' do
+    let(:meta) do
+      <<-HEREDOC
+      <project name="#{project.name}">
+      <title/><description/>
+      </project>
+      HEREDOC
+    end
+
+    let(:project_updater) { ::MetaControllerService::ProjectUpdater.new(project: project, request_data: Xmlhash.parse(meta)) }
+
+    describe '#new' do
+      it { expect(project_updater).not_to be_nil }
```

Comment:
This is testing that the constructor works? I don't really see why it's needed...

Reply:
The constructor is written by us. So it makes sense to test it. But `not_to be_nil` could be anything. I would either be more specific or drop the test.

Reply:
Check that the created instance is of the expected class (the [project updater](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/services/meta_controller_service/project_updater.rb#L3) seems to handle this and that the request data was assigned.

Reply:
I removed it completely @dmarcoux  and @bgeuken